exercises/two-fer/instructions.md

```diff
@@ -0,0 +1,24 @@
+# Instructions
+
+Your task is to determine what you will say to your friend as you hand them their cookie.
+
+If your friend is named Do-yun, then you will say:
+
+```text
+One for Do-yun, one for me.
+```
+
+On the other hand, if for some reason you can't remember your friend's name just then, you'll say _you_ instead of their name:
```

Comment:
Can you add some rationale pls?

Reply:
I'm guessing it doesn't feel good to forget a friend's name?

Maybe we could play on the thing where in some places it's more common to say _you_ when speaking directly to someone, whereas in others (e.g. Korea) it's common to use someone's name?

What other possible reasons can you think of for using _you_ instead of the name?

Reply:
I like the suggestion about giving the cookie to a stranger whose name you don't know, if the friend doesn't like a cookie. I've updated the text to use that as the reason.